modules/optics/arrow-optics/src/test/kotlin/arrow/optics/instances/MapInstanceTest.kt

```diff
@@ -71,7 +73,7 @@ class MapInstanceTest : UnitSpec() {
     ))
 
     testLaws(OptionalLaws.laws(
-      optional = MapK.index<String, Int>().index(Gen.string().generate()),
+      optional = MapK.index<String, Int>().index(""),
```

Comment:
Did these laws fail, and only pass for empty string? @nomisRev ?

Reply:
No, they don't only pass for `empty string`. Ideally, it would use a generator suitable for generating keys.